verto/errors/PanelMissingTitleError.py

```diff
@@ -0,0 +1,18 @@
+from verto.errors.Error import Error
+
+
+class PanelMissingTitleError(Error):
+    '''Exception raised when a panel is missing a title.
+
+    Attributes:
+        tag: tag which was not matched
+        block: block where tag was not matched
+        argument: the argument that was not found
+        message: explanation of why error was thrown
```

Comment:
This exception is so specific, that the message could be hardcoded here instead of passed in.